web/main/tasks.py

```diff
@@ -43,10 +44,12 @@ def generate_pdf(
     page.on("console", lambda msg: logger.warning(f"From browser console: {msg}"))
     expect(page.locator(selector)).to_be_visible(timeout=timeout)
     pdf = page.pdf()
-    output_file.write_bytes(pdf)
-
+    storage = get_s3_storage(bucket_name=os.environ["PDF_EXPORT_BUCKET"])
```

Comment:
Suggestion: for consistency, can you make this a Django setting that is set to `os.environ["PDF_EXPORT_BUCKET"]` by default? That will make it easier for us to override in tier-level settings.py